### ISSUE TITLE
Limits on student employee work hours on holidays or closure days

### ISSUE THREAD

**Issue by pattonsgirl** [wrightedu/TA-Handbook#12]
Per November update from HR
a.    As with most other types of university employees, student employees are not expected to work when the university is closed for holidays, winter break, inclement weather or other closure reasons. Some exceptions apply for essential duties, in which case the supervisor must be available to direct and verify the student employee work on the holiday or closure day:

b.    Employee work on holidays or closure days can only be permitted at the supervisor’s direction. Examples of holiday or closure day work that is essential include:

i.        Snow removal duties that the supervisor requests be performed.
ii.        Duties assisting an ongoing faculty research project that, by research protocol, require daily attention during holiday or closure day(s).
iii.        Tasks essential to the set-up, implementation, or tear-down for events that occur on the holiday or closure day and are scheduled by the supervisor.
iv.        Personal Assistant tasks providing support to Office of Disability Services clients, scheduled by the supervisor to cover needs on the holiday or closure day.

TLDR: on days with the University is **closed**, may not be able to log hours.  On no-classes held, University open, type days, hours are valid.

**Comment by pattonsgirl** [comment]
d.    Please ensure that student employees are aware of the [university holiday closure schedule](http://www.wright.edu/human-resources/holiday-schedule), [Winter Break dates](http://www.wright.edu/human-resources/winter-break-and-leave), and other regular or emergency closure schedules, with instructions that they do not work on those days nor do they report on their time sheet any hours for those days.
